rest/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "learning",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon server.js",
```

Comment:
if you have devStart and its for dev environment, then **start** should be for production environment which is not for developing, so there is no need to run nodemon. Change to **node server.js**